pkg/testutil/nerdtest/platform/platform_linux.go

```diff
@@ -0,0 +1,33 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package platform
+
+import (
+	"github.com/containerd/nerdctl/v2/pkg/rootlessutil"
+	"github.com/containerd/nerdctl/v2/pkg/testutil"
+)
+
+func DataHome() (string, error) {
+	return rootlessutil.XDGDataHome()
+}
+
+var (
+	RegistryImageStable = testutil.RegistryImageStable
+	RegistryImageNext   = testutil.RegistryImageNext
+	KuboImage           = testutil.KuboImage
+	DockerAuthImage     = testutil.DockerAuthImage
+)
```

Comment:
Because we are stuck in a bad situation right now with a mix of `_platform_test.go` files (which should disappear) and `Skip(platform)` tests, that must have the corresponding helpers available.

This will get cleaned-up later on, but rn a level of indirection is necessary (or alternatively add a bunch of stuff in the legacy files).